typescript/formsApps.d.ts

```diff
@@ -501,6 +501,7 @@ export type FormsAppConfiguration<
   isGoogleLoginSupported: boolean
   isClientLoggingEnabled: boolean
   recaptchaPublicKey: string
+  googleMapsIntegrationKeyId?: string
```

Comment:
Whoops, this property should actually be what you had it as before. `googleMapsIntegrationKeyId` should be the property on the `FormsApp` type.